packages/x-charts/src/XAxis/xAxisClasses.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  unstable_generateUtilityClass as generateUtilityClass,
+  unstable_generateUtilityClasses as generateUtilityClasses,
+} from '@mui/utils';
+
+export interface XAxisClasses {
```

Comment:
Would it make sense to have the same classes for the X and Y axes (to avoid duplication) and a class `directionX` / `directionY` to differentiate the two ?

Reply:
Nice ide, I update it such that roo get the direction and the position:

For example bottom x axis will get `MuiAxis-root MuiAxis-directionX MuiAxis-bottom` classes

Reply:
Actually, I wonder if `MuiAxis-directionX` is needed, since we have `MuiAxis-bottom`

In which scenario would you need `directionX` ?

Reply:
To get the same style on top and bottom without having to care about the position

For example you you show both axis, you could customize bottom and top ticks the same way, with `directionX` classe and remove only top ticks labels with `top` classe